doc/source/languages.rst

```diff
@@ -0,0 +1,67 @@
+Languages
+=========
+
+A program can be a very useful tool, but many times one is expected
+to give access to it to many people who are not native English 
+speakers.
+
+Libreant makes use of a specific module of Flask (Flask-Babel) which
```

Comment:
I made the correction you suggested.

On 27/02/2015 17:56, BoySka wrote:
>> @@ -0,0 +1,67 @@
>> +Languages
>> +=========
>> +
>> +A program can be a very useful tool, but many times one is expected
>> +to give access to it to many people who are not native English 
>> +speakers.
>> +
>> +Libreant makes use of a specific module of Flask (Flask-Babel) which
> is this paragraph useful, for a non-developer?
>
It is just an introduction. I think it is just the right place to
introduce the module we used.